pkg/networking/security_group_resolver_test.go

```diff
@@ -205,13 +258,34 @@ func Test_defaultSecurityGroupResolver_ResolveViaNameOrID(t *testing.T) {
 			wantErr: errors.New("Describe.Error: unable to describe security groups"),
 		},
 		{
-			name: "unable to resolve all security groups",
+			name: "unable to resolve all security group ids",
 			args: args{
 				nameOrIDs: []string{
-					"sg group one",
```

Comment:
can we keep a negative test case for the mix name&ids ?

Reply:
I pushed https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/3775/commits/12a7953a3112c0478a3f5178bf16b5650910e14d to add the negative test for mix name & ids as well as improve error reporting a little bit.